pipeline/jobs/jobdsl.groovy

```diff
@@ -13,8 +13,7 @@ pipelineJob('aws-devsecops-workshop') {
         git {
           branch('master')
           remote {
-            credentials('jenkins')
-            url('git@github.com:stelligent/aws-devsecops-workshop.git')
+            url('https://github.com/stelligent/aws-devsecops-workshop.git')
```

Comment:
https is used, as this is a public repo.  this is in preparation for having jenkins come up via cloudinit, instead of manually prebaked ami.  there will be no need to conf jenkins creds on bootstrap this way.